<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt</class>
 <widget class="QMainWindow" name="qt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>468</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Camera Display/Serial Communication GUI v1.2</string>
  </property>
  <property name="windowOpacity">
   <double>0.990000000000000</double>
  </property>
  <property name="statusTip">
   <string>2022</string>
  </property>
  <property name="whatsThis">
   <string>Camera Display/Serial Communication Program 2022</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>1677799</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="MainScreen">
        <attribute name="title">
         <string>Main Screen</string>
        </attribute>
        <widget class="QPushButton" name="pushButton_23">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>340</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>START</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_24">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>340</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_25">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>340</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>SAVE</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_26">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>150</y>
           <width>161</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>ANALYSIS</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_27">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>90</y>
           <width>161</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>TARGET</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_28">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>210</y>
           <width>161</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>WAVELENGTH</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_29">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>270</y>
           <width>161</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>PHYSICOGICAL DATA</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_3">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>30</y>
           <width>71</width>
           <height>51</height>
          </rect>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_4">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>30</y>
           <width>71</width>
           <height>51</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_33">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>0</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>FPS IN</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_48">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>0</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>FPS OUT</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_49">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>421</width>
           <height>281</height>
          </rect>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
        <!-- <widget class="QPushButton" name="pushButton_30">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>340</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Desktop Camera</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_31">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>340</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Off Camera</string>
         </property>
        </widget> -->
       </widget>
       <widget class="QWidget" name="SerialMonitor">
        <attribute name="title">
         <string>Serial Monitor</string>
        </attribute>
        <widget class="QTextEdit" name="textEdit_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>501</width>
           <height>241</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>280</y>
           <width>421</width>
           <height>81</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>290</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>SEND</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>240</y>
           <width>251</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Gathering / Sending Data (Rx/Tx)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>250</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Send DATA:</string>
         </property>
        </widget>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>0</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Port Avaliable:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="formLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>30</y>
           <width>171</width>
           <height>21</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Port Status</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Not Connected</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>50</y>
           <width>197</width>
           <height>191</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Connection Options:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>STOP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>START</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Connection Percentage Status:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="About">
        <attribute name="title">
         <string>About</string>
        </attribute>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>661</width>
           <height>301</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:16px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji'; font-size:8pt; color:#24292f; background-color:#ffffff;&quot;&gt;Run&lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji'; font-size:8pt; color:#24292f;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace'; font-size:8pt; color:#24292f;&quot;&gt;capture_display.py&lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji'; font-size:8pt; color:#24292f;&quot;&gt; from &lt;/span&gt;&lt;span style=&quot; font-family:'ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace'; font-size:8pt; color:#24292f;&quot;&gt;.\examples&lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji'; font-size:8pt; color:#24292f;&quot;&gt; You need to set the proper config file in the program. You should not need to edit python files in capture or streamer folder.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:24px; margin-bottom:16px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:125%; background-color:#ffffff;&quot;&gt;&lt;a name=&quot;user-content-example-programs&quot;&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji'; font-size:8pt; font-weight:600; color:#24292f;&quot;&gt;E&lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji'; font-size:8pt; font-weight:600; color:#24292f;&quot;&gt;xample Programs&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:16px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji'; font-size:8pt; font-weight:600; color:#24292f;&quot;&gt;Display&lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji'; font-size:8pt; color:#24292f;&quot;&gt;:&lt;br /&gt;In general display should occur in main program. OpenCV requires waitkey to be executed in order to update the display and limits update rate to about 50-90 fps.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:16px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji'; font-size:8pt; font-weight:600; color:#24292f;&quot;&gt;Queue&lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji'; font-size:8pt; color:#24292f;&quot;&gt;: Data transfer between the main program and capture and storage threads.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Help">
        <attribute name="title">
         <string>Help</string>
        </attribute>
        <widget class="QTextBrowser" name="textBrowser_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>701</width>
           <height>371</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Contact &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;University of Arizona&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAna_Ekran">
   <property name="text">
    <string>Ana Ekran</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
